test/deletes.spec.js

```diff
@@ -0,0 +1,23 @@
+var _ = require('lodash');
+var should = require('should');
+var $http = require('http-as-promised');
+
+var seeder = require('./seeder.js');
+
+describe("deletes", function () {
+
+    var config;
+    beforeEach(function () {
+        config = this.config;
+        return seeder(this.app).dropCollectionsAndSeed().then(function (_ids) {
```

Comment:
If you know what collections ('people' in this case) are used, would be nice to see them called out here, for speed and so that it becomes the default way people interact with your code.